src/MUIDataTable.js

```diff
@@ -936,6 +936,8 @@ class MUIDataTable extends React.Component {
             filterList[index] = value === '' ? [] : value;
             break;
           case 'dropdown':
+            filterList[index] = value;
```

Comment:
Actually, do we need the `default` option anymore? `dropdown` used to fall into it, but now nothing will except invalid options, right? We could replace it with a warning or something.

Reply:
Yes, the type "textField" falls through and gets set there.